components/tabs/Tabs.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC, useState, ReactElement } from "react";
+import Tab from "./Tab";
+import { TabList } from "./Tabs.styled";
+import { TabProps } from "./Tab";
+interface Props {
+  children: Array<ReactElement<TabProps>>;
```

Comment:
Why is this redefined here? `React.FC` does include the `children` prop. If it is done because TS is complaining about the children mapping that follows, you can use `React.Children.map` which is an helper method on React that won't make TS complain and allows you to map through childrens

Reply:
I think this is actually in part *saying* what children need to be a part of this component. So we don't want *any* children but specific ones with certain attributes.